plugins/woocommerce/includes/class-wc-template-loader.php

```diff
@@ -159,6 +159,7 @@ private static function has_block_template( $template_name ) {
 	 * @since  3.0.0
 	 * @since  5.5.0 If a block template with the same name exists, return an
 	 * empty string.
+	 * @since  6.1.0 It checks custom product taxonomies
```

Comment:
```suggestion
	 * @since  6.2.0 It checks custom product taxonomies
```

This should be 6.2.0 now I think.